README.md

```diff
@@ -26,15 +26,10 @@ original ones as well!).
 ```json
 {
   "rules": {
-    "babel/generator-star-spacing": 1,
     "babel/new-cap": 1,
-    "babel/array-bracket-spacing": 1,
     "babel/object-curly-spacing": 1,
-    "babel/object-shorthand": 1,
-    "babel/arrow-parens": 1,
     "babel/no-await-in-loop": 1,
     "babel/flow-object-type": 1,
```

Comment:
This can be removed as well. It's now https://github.com/gajus/eslint-plugin-flowtype#eslint-plugin-flowtype-rules-object-type-delimiter.